Document stopTaskCount supervisor property (#18165)
apache/druid: docs/ingestion/supervisor.md

|`lateMessageRejectionPeriod`|ISO 8601 period|Configures tasks to reject messages with timestamps earlier than this period before the task was created. For example, if this property is set to `PT1H` and the supervisor creates a task at `2016-01-01T12:00Z`, Druid drops messages with timestamps earlier than `2016-01-01T11:00Z`. This may help prevent concurrency issues if your data stream has late messages and you have multiple pipelines that need to operate on the same segments, such as a streaming and a nightly batch ingestion pipeline. You can specify only one of the late message rejection properties.|No||
|`earlyMessageRejectionPeriod`|ISO 8601 period|Configures tasks to reject messages with timestamps later than this period after the task reached its task duration. For example, if this property is set to `PT1H`, the task duration is set to `PT1H` and the supervisor creates a task at `2016-01-01T12:00Z`, Druid drops messages with timestamps later than `2016-01-01T14:00Z`. Tasks sometimes run past their task duration, such as in cases of supervisor failover.|No||

> |`stopTaskCount`|Integer|Limits the number of ingestion tasks Druid can cycle at any given time. If not set, Druid can cycle all tasks at the same time. If set to a value less than `taskCount` your cluster needs fewer available slots to run the supervisor, so you can save costs by scaling down your ingestion tier. See [stopTaskCount](#stoptaskcount) for more information.|No|`taskCount` value|

Reviewer: I know this links to a section, but to me it feels worth explicitly mentioning even here that this increases overall cycle time even though it may be obvious, because if you don't read the other section or read between the lines you might miss that reducing ingestion cost isn't free and operators probably shouldn't use this setting unless they know what they are doing. Maybe something like:
```
so you can save costs by scaling down your ingestion tier in exchange for slower cycle time and possible lag.
```

Author: updated.

> |`stopTaskCount`|Integer|Limits the number of ingestion tasks Druid can cycle at any given time. If not set, Druid can cycle all tasks at the same time. If set to a value less than `taskCount` your cluster needs fewer available slots to run the supervisor, so you can save costs by scaling down your ingestion tier in exchange for slower cycle time and possible lag. See [stopTaskCount](#stoptaskcount) for more information.|No|`taskCount` value|

Reviewer 2: ```suggestion
|`stopTaskCount`|Integer|Limits the number of ingestion tasks Druid can cycle at any given time. If not set, Druid can cycle all tasks at the same time. If set to a value less than `taskCount`, your cluster needs fewer available slots to run the supervisor. You can save costs by scaling down your ingestion tier, but this can lead to slower cycle times and lag. See [stopTaskCount](#stoptaskcount) for more information.|No|`taskCount` value|
```
